recipe_scrapers/nrkmat.py

```diff
@@ -0,0 +1,50 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._exceptions import SchemaOrgException
+from ._grouping_utils import group_ingredients
+
+
+class NRKMat(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "nrk.no"
+
+    def author(self):
+        return (
+            self.soup.find("span", {"class": "caption-cite"})
+            .getText()
+            .removeprefix("Kilde: ")
+        )
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def yields(self):
+        try:
+            return self.schema.yields()
+        except SchemaOrgException:
+            return ""
```

Comment:
I'd suggest allowing the exception to be raised here - in other words, removing the `try...catch` logic.  Unless there's a reason to handle errors this way for this website?

Reply:
Not all recipes specify a number of servings, so I thought this would be the best way to handle it. If you have any other ideas I'd love to hear them:)
I see now however that similar try except blocks in other files return None, which makes more sense than an empty string, so I'll change it to that for now.

Reply:
Yep, no better suggestions than what you suggest yourself :)  Thanks for looking into that!